app/notify_admin_test.go

```diff
@@ -141,7 +141,125 @@ func Test_SendNotifyAdminPosts(t *testing.T) {
 		require.Equal(t, "1 member of the test workspace has requested starting the Enterprise trial for access to: ", post.Message)
 	})
 
-	t.Run("error when trying to send post before end of cool off period", func(t *testing.T) {
+	t.Run("successfully send install plugin post", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+
+		// some some notifications
+		_, appErr := th.App.SaveAdminNotifyData(&model.NotifyAdminData{
+			UserId:          th.BasicUser.Id,
+			RequiredPlan:    "github",
```

Comment:
Nit: Add `github` to https://github.com/mattermost/mattermost-server/blob/master/model/plugin_constants.go as `PluginIdGithub`